editor/src/clj/editor/mesh.clj

```diff
@@ -148,11 +150,14 @@
             :deps dep-build-targets})])))
 
 (g/defnk produce-gpu-textures [_node-id samplers gpu-texture-generators]
-  (into {} (map (fn [unit-index sampler {tex-fn :f tex-args :args}]
-                  (let [request-id [_node-id unit-index]
-                        params     (material/sampler->tex-params sampler)
-                        texture    (tex-fn tex-args request-id params unit-index)]
-                    [(:name sampler) texture]))
+  (into {} (map (fn [unit-index sampler gpu-texture-generator]
+                  (let [gpu-texture (texture-util/generate-gpu-texture gpu-texture-generator)]
```

Comment:
I couldn't figure out how to repro it, but `texture-util/generate-gpu-texture` asserts that its argument is `gpu-texture-generator?`. Is it possible that a cubemap will end up there? If so, cubemap produces a value that satisfies `cubemap-gpu-texture-generator?`, and this is NOT the same as `gpu-texture-generator?`.